Dockerfile

```diff
@@ -0,0 +1,37 @@
+# This docker image is for integration testing only.
+
+FROM golang:1.12-buster
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV TEST_BINARY=/test/trellis-cli
+ENV TEST_DUMMY=/test/dummy
+
+WORKDIR /app
+
+# CircleCI
+# https://circleci.com/docs/2.0/custom-images/
+RUN apt-get -q update && \
+    apt-get install -q -y --no-install-recommends git openssh-client openssh-server tar gzip && \
+    apt-get clean && apt-get -y autoremove && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Ansible
+# https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#latest-releases-via-apt-debian
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367 && \
+    echo "deb http://ppa.launchpad.net/ansible/ansible-2.7/ubuntu trusty main" | tee -a /etc/apt/sources.list && \
+    apt-get -q update && \
+    apt-get install -q -y --no-install-recommends ansible && \
+    apt-get clean && apt-get -y autoremove && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Trellis
```

Comment:
Maybe in the future we could use trellis-cli itself to create a new project which takes care of this (and would test trellis-cli itself)!